src/settingsFields.component.js

```diff
@@ -1,5 +1,5 @@
 import i18n from '@dhis2/d2-i18n'
-import { Button, Card } from '@dhis2/ui'
+import { Button, Card, CenteredContent, CircularLoader } from '@dhis2/ui'
```

Comment:
More `@dhis2/ui` components here.

Reply:
@HendrikThePendric from the looks of it this already has mixed UI, right?